pkg/primitives/statistics.go

```diff
@@ -60,9 +60,21 @@ func NewStatistics(total gridtypes.Capacity, storage provision.Storage, reserved
 	}
 }
 
+// deploymentsCount Gets count of deployments
+func (s *Statistics) deploymentsCount() (int, error) {
+	_, deps, _, err := s.storage.Capacity()
+	return len(deps), err
+}
+
+// workloadsCount Gets count of workloads for each deployment
+func (s *Statistics) workloadsCount() (int, error) {
+	_, _, workloads, err := s.storage.Capacity()
+	return len(workloads), err
+}
+
```

Comment:
the `storage.Capacity()` is kinda a heavy call. And we try to avoid it as much as possible. So why not create a single call to it instead that return the counts instead 
```
return len(deps), workloads, err 
```